src/accidental.ts

```diff
@@ -90,7 +110,7 @@ export class Accidental extends Modifier {
           dblSharpLine: true,
           numAcc: 0,
           width: 0,
-        });
+        } as Line);
```

Comment:
Which members of `line` are missing? Perhaps these should be made optional with ?. Using `as` we can build objects inconsistent with the type.